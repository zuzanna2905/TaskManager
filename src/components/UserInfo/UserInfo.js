import React from 'react';

const UserInfo = ({name, areTasks}) => {
  if(areTasks){
  return (
    <div className='f3 mv1'>
      Hi {name}!
      Your tasks
    </div>
  )}
  else {
    return (
    <div className='f3 mv1'>
      Hi {name}!
      You have no tasks
    </div>
  )}
}

export default UserInfo;