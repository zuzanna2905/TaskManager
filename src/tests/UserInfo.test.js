import {shallow} from 'enzyme';
import React from 'react';
import UserInfo from '../components/UserInfo/UserInfo';

it('renders UserInfo without crashing', () => {
    const initialProps = {
        name: 'Ela',
        areTasks: true
    }
    let  wrapper = shallow(<UserInfo {...initialProps}/>);
    expect(wrapper).toMatchSnapshot();
});

it('renders UserInfo no task', () => {    
    const initialProps2 = {
        name: 'Ela',
        areTasks: false
    }
    let wrapper2 = shallow(<UserInfo {...initialProps2}/>);
    expect(wrapper2).toMatchSnapshot();
})