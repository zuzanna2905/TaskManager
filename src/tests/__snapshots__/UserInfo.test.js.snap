// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UserInfo no task 1`] = `
<div
  className="f3 mv1"
>
  Hi 
  Ela
  ! You have no tasks
</div>
`;

exports[`renders UserInfo without crashing 1`] = `
<div
  className="f3 mv1"
>
  Hi 
  Ela
  ! Your tasks
</div>
`;
